src/pages/Application/Loaded.tsx

```diff
@@ -0,0 +1,147 @@
+import { PropsWithChildren } from "react";
+import { Link } from "react-router-dom";
+import { ApplicationDetails } from "types/aws";
+import { useModalContext } from "contexts/ModalContext";
+import ExtLink from "components/ExtLink";
+import Icon from "components/Icon";
+import { appRoutes } from "constants/routes";
+import Container from "./Container";
+import Prompt from "./Prompt";
+
+const NA = "Not provided";
+
+export default function Loaded(props: ApplicationDetails) {
+  const { showModal } = useModalContext();
+  const { ContactPerson: c, Registration: r, WiseRecipient: w } = props;
+  const doc = r.Documentation;
+
+  const review = (verdict: "approve" | "reject") => () => {
+    showModal(Prompt, { verdict, uuid: c.PK, orgName: r.OrganizationName });
+  };
+
+  const prevVerdict =
+    r.RegistrationStatus === "Active" || r.RegistrationStatus === "Rejected"
+      ? r.RegistrationStatus
+      : null;
+
+  return (
+    <>
+      <h3 className="text-lg">{r.OrganizationName}</h3>
+      {prevVerdict && (
+        <div
+          className={`${
+            prevVerdict === "Active" ? "bg-green" : "bg-red"
+          } text-white px-2 py-1 text-xs font-work uppercase rounded justify-self-start -mt-3 lg:-mt-6`}
+        >
+          {prevVerdict === "Active" ? "Approved" : "Rejected"}
+        </div>
+      )}
+      <div className="flex max-sm:flex-col gap-x-4">
+        <span className="text-sm font-semibold uppercase">Application ID:</span>
+        <span className="uppercase text-sm font-work">{r.PK}</span>
+      </div>
+      <div className="flex max-sm:flex-col gap-x-4 -mt-2 lg:-mt-4">
+        <span className="text-sm font-semibold uppercase">Date submitted:</span>
+        <span className="uppercase text-sm font-work">
+          {new Date(r.RegistrationDate).toLocaleDateString()}
+        </span>
+      </div>
+
+      <Container title="Endowment application">
+        <div className="grid sm:grid-cols-[auto_auto_1fr]">
+          {doc.DocType === "FSA" ? (
+            <Row label="Registration No.">{doc.RegistrationNumber}</Row>
+          ) : (
+            <Row label="EIN">{doc.EIN}</Row>
+          )}
+          <Row label="HQ Country">{r.HqCountry}</Row>
+          <Row label="Countries active in">
+            {r.ActiveInCountries.join(", ")}
+          </Row>
+          <Row label="UN SDG">{r.UN_SDG.join(", ")}</Row>
+          <Row label="Contact name">{c.FirstName + " " + c.LastName}</Row>
+          <Row label="Contact email">{c.Email}</Row>
+          {doc.DocType === "FSA" && (
+            <>
+              <Row label="Contact national ID">
+                <DocLink url={doc.ProofOfIdentity.publicUrl} />
+              </Row>
+              <Row label="Charity registration doc">
+                <DocLink url={doc.ProofOfRegistration.publicUrl} />
+              </Row>
+              <Row label="Fiscal sponsorship agreement">
```

Comment:
Maybe better we hide this row completely if `doc.SignedFiscalSponsorshipAgreement == null`, i.e. it wasn't uploaded? @SovereignAndrey

Reply:
I think it is OK to leave as is for now. Reviewer would know pretty easily from HQ country.